internal/pkg/manager/spod/spod_controller.go

```diff
@@ -587,6 +587,7 @@ func (r *ReconcileSPOd) getConfiguredSPOd(
 
 	templateSpec.Tolerations = cfg.Spec.Tolerations
 	templateSpec.Affinity = cfg.Spec.Affinity
+	templateSpec.ImagePullSecrets = cfg.Spec.ImagePullSecrets
```

Comment:
I'm wondering if we should apply this to all other images we pull as well. :thinking:

Reply:
Do you mean in the webhook binding?

Reply:
Yeah for example, we also use kube rbac proxy and things like that but their manifests are mostly static.

Reply:
I configured it also in the webhook default deployment. Are there any other place which we should consider?

Reply:
The daemon probably.

Reply:
I am a bit confused. Isn't the spod the daemon? I think the other resources get deployed declaratively using manifests.